modules/nw-endpoint-route53.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-government-region.adoc
+
+[id="nw-endpoint-route53_{context}"]
+= Endpoint configuration for route53
+
+The `route53` endpoint URL must use regions, `us-east-1`, `us-west-1`, or no region because Route 53 is not a regionalized service. The `us-east-1` region is required for the `tagging` endpoint URL to get the hosted zone, `route 53`, of the Elastic Load Balancing (ELB).
+
+[IMPORTANT]
+====
+Private, disconnected installs are not supported for AWS GovCloud when installing in the `us-gov-east-1` region.
+====
+
+.Example `route53` configuration
+[source,yaml]
+----
+platform:
+  aws:
+    region: us-gov-west-1
+    serviceEndpoints:
+    - name: ec2
+       url: https://ec2.us-gov-west-1.amazonaws.com
+    - name: elasticloadbalancing
+       url: https://elasticloadbalancing.us-gov-west-1.amazonaws.com
+    - name: route53 # Route 53 is not regionalized, so us-east-1 or no region.
```

Comment:
I suggest removing `# Route 53 is not regionalized, so us-east-1 or no region.`.